crates/common/flockfile/src/unix.rs

```diff
@@ -1,5 +1,6 @@
 use nix::fcntl::flock;
 use nix::fcntl::FlockArg;
+use nix::unistd::{write, Pid};
```

Comment:
Please run `cargo +nightly fmt` as we are using `imports_granularity = "Item"` i.e. a single import per line.